File: 3065/main.go
package main

import (
	"fmt"
)

func main() {
	fmt.Println(minOperations([]int{2,11,10,1,3}, 10))
}

func minOperations(nums []int, k int) int {
	ans := 0 
	for i := range nums {
		if nums[i] < k {
			ans++
		}
	}
	return ans
}

File: 3065/main.rs
struct Solution{}

impl Solution {
    pub fn min_operations(nums: Vec<i32>, k: i32) -> i32 {
        let mut ans = 0;
        for i in 0..nums.len() {
            if nums[i] < k {
                ans += 1;
            }
        }
        ans
    }
}

fn main() {
    println("{:#?}", Solution::min_operations(vec![1,1,2,4,9], 1));
}